tests/sentry/event_manager/test_event_manager.py

```diff
@@ -1845,9 +1848,10 @@ def test_save_issueless_event(self):
             tsdb.get_sums(tsdb.models.project, [self.project.id], event.datetime, event.datetime)[
                 self.project.id
             ]
-            == 1
+            == 0
```

Comment:
These counts don't make sense anymore, since we no longer store events and transactions in the same table

Reply:
Just curious about this. Before the count was 1, was this some odd side-effect or hack that was introduced before when storing transactions?

Reply:
We used to store errors and transactions in the same table, and used `event.type` to differentiate between them. `get_sums(tsdb.models.project, ...)` reads from the events dataset when we use `SnubaTSDB`. So when it checks the events dataset, there are no transactions in that table (there would have been when this test was first written).

The test used to work with a value of 1 because we were using `RedisTSDB`, which gets incremented here: https://github.com/getsentry/sentry/blob/253d97f4e447186a0f9194539bb02b0b225ebe06/src/sentry/event_manager.py#L1004

(As a side note... I think that these incr of the `models.project` model are actually unnecessary)